package.json

```diff
@@ -91,5 +91,43 @@
             "eslint --check",
             "prettier --check"
         ]
+    },
+    "dependencies": {
+        "bluebird": "3.7.2",
+        "bootstrap": "3.3.7",
+        "bootstrap-slider": "10.6.2",
+        "bowser": "1.0.0",
+        "corejs-typeahead": "1.0.1",
+        "d3": "3.5.6",
+        "datatables": "1.10.9",
+        "datatables.net-bs": "2.1.1",
+        "datatables.net-buttons-bs": "1.4.2",
+        "dropzone": "5.7.0",
+        "file-saver": "1.3.4",
+        "font-awesome": "4.7.0",
+        "google-code-prettify": "1.0.5",
+        "handlebars": "4.0.5",
+        "jquery": "^2.2.4",
+        "jquery-nearest": "1.3.1",
+        "jquery-ui": "^1.12.1",
+        "jquery.tipsy": "1.0.3",
+        "js-yaml": "3.3.1",
+        "kbase-common-js": "https://github.com/kbase/kbase-common-js/archive/refs/tags/v2.16.0.tar.gz",
+        "kbase-sdk-clients-js": "https://github.com/eapearson/kbase-sdk-clients-js/archive/refs/tags/v0.5.1.tar.gz",
+        "kbase-service-clients-js": "https://github.com/kbase/kbase-service-clients-js/archive/refs/tags/v3.3.4.tar.gz",
+        "kbase-ui-plugin-catalog": "https://github.com/kbase/kbase-ui-plugin-catalog/archive/refs/tags/v2.2.5.tar.gz",
```

Comment:
These aren't registered on NPM, but these versions mostly match and have package.json files that NPM needs.